src/app/review-item/review-item.module.ts

```diff
@@ -25,6 +25,7 @@ import { UserExperienceBubbleListModule } from '@shared/user-experience-bubble-l
 import { WatsonComponent } from './components/watson/watson.component';
 import { IvyCarouselModule } from 'angular-responsive-carousel';
 import { BreadcrumbModule } from '../shared/breadcrumb/breadcrumb.module';
+import { PipesModule } from '@shared/pipes/pipes.module';
```

Comment:
You have two different ways of importing the module:

`import { PipesModule } from '@shared/pipes/pipes.module';`
`import { PipesModule } from '../pipes/pipes.module';`

Either is fine IMO, I would just stick with one.